README.md

```diff
@@ -121,7 +121,7 @@ For example, installing [`oneTBB`](https://github.com/oneapi-src/oneTBB) on Linu
 ```bash
 TBB_VERSION="2021.1.1"
 
-wget https://github.com/oneapi-src/oneTBB/releases/download/v2021.1.1/oneapi-tbb-$TBB_VERSION-lin.tgz
+wget https://github.com/oneapi-src/oneTBB/releases/download/v$TBB_VERSION/oneapi-tbb-$TBB_VERSION-lin.tgz
```

Comment:
I think the standard thing here is to use curly braces like 

```suggestion
wget https://github.com/oneapi-src/oneTBB/releases/download/v{TBB_VERSION}/oneapi-tbb-$TBB_VERSION-lin.tgz
```

It makes it a little more obvious it's user supplied

Reply:
Sounds good. I think you mean to replace `$TBB_VERSION` with `${TBB_VERSION}` in all locations; right? I can't commit your suggestion because it's missing `$`.

Reply:
Yes! Just updated the suggestion